sub/osd.c

```diff
@@ -389,6 +392,12 @@ struct sub_bitmap_list *osd_render(struct osd_state *osd, struct mp_osd_res res,
     if (!(draw_flags & OSD_DRAW_SUB_FILTER))
         osd->want_redraw_notification = false;
 
+    double elapsed = MP_TIME_NS_TO_MS(mp_time_ns() - start_time);
+    bool slow = elapsed > 5;
```

Comment:
it might be more useful if this actually scaled with how much frame time budget we have for our current configuration. thoughts? it's fine if that's too much work to implement though

Reply:
That would be ideal, else it is really hard to know what value is acceptable. But this information is not easily available here.